Sound/Tidal/Stream.hs

```diff
@@ -218,6 +218,17 @@ infixl 1 |/|
 (|/|) :: ParamPattern -> ParamPattern -> ParamPattern
 (|/|) = mergeNumWith (div) (/)
 
+{- | These are shorthand for merging lists of patterns with @#@, @|*|@, @|+|@,
```

Comment:
questions only: wouldn't it be more general to add a very short definition of `foldl` which you can combine with any function? something like `(^ #)` Well, I see that this doesn't type itself so well. 

Or, alternatively, why not only `##` but three of them?